core/server/data/migrations/versions/4.0/26-add-cascade-on-delete.js

```diff
@@ -0,0 +1,75 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addForeign, dropForeign} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    if (knex.client.config.client !== 'sqlite3') {
+        return logging.warn('Skipping  - database is not SQLite3');
```

Comment:
Can you make this something like "Skipping adding 'on cascade delete' - database is not SQLite3"? Just so it's even more explicit what we're early returning from 🙂

Reply:
good point! done ✅